### PR TITLE
Feature unzip

### PR DESCRIPTION
**Pull request by Vermouth02**
extract .zip files

### CODE REVIEW

**Comment by sirateek** [comment]
Since I updated the branch name from `main` to `master`. I noticed the misconfiguraton in the workflow file.  This cause the workflow not autumatically run due to the wrong target branch name.

I fixed it already in master and I also merged it into your branch. So you can continue your work as normal 👍

File: lib/file_management/extract.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+### if have any suggestions or problem, you can direct message to me
+"""
+
+import os
+import zipfile
+from file_management_lib import DirManagement, WorkEditor
+
+
+def check_filename_draft(filename: list,draft: list) -> bool:
```
Comment (vitvara) [inline lib/file_management/extract.py:16]: what type of draft

File: lib/file_management/extract.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        else:
+            reminder += i
+    list_filename = filename.split("_")
+    if not check_filename_draft(list_filename,key):
+        print("Invalid file name " + filename)
+        return False
+    else:
+        return True
+
+def unzipfile(path: str):
```
Comment (vitvara) [inline lib/file_management/extract.py:52]: path of working directory?

File: lib/file_management/extract.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    """
+    'path: (str)' is directory name that you want this function to extract files and create folders in this
+    You should to change backslash to sla for prevebt backslash error
+    The floder's name where files extracted is same as the zip file's name.
+    """
+    listfile = os.listdir(path)
+    create_dir = DirManagement().create_dir
+    for i in listfile:
```
Comment (vitvara) [inline lib/file_management/extract.py:60]: can you change `i` into something that other people can understand?

**Review by vitvara — APPROVED** [review]
nicework

**Pull request merged**